Source/WebCore/platform/mediastream/ios/AVAudioSessionCaptureDevice.h

```diff
@@ -36,14 +36,21 @@ namespace WebCore {
 
 class AVAudioSessionCaptureDevice : public CaptureDevice {
 public:
-    static AVAudioSessionCaptureDevice create(AVAudioSessionPortDescription *deviceInput, AVAudioSessionPortDescription *defaultInput);
+    static AVAudioSessionCaptureDevice createInput(AVAudioSessionPortDescription *deviceInput, AVAudioSessionPortDescription *defaultInput);
+    static AVAudioSessionCaptureDevice createOutput(AVAudioSessionPortDescription *deviceOutput, AVAudioSessionPortDescription *defaultOutput);
     virtual ~AVAudioSessionCaptureDevice() = default;
 
     AVAudioSessionCaptureDevice isolatedCopy() &&;
 
+    bool isBuiltin() const;
+    bool isLine() const;
```

Comment:
Nit: `isLine` is a confusing name. It might be better to have different methods for line-in and line-out, or if it is better to have one method rename to something line `isLineInOrOut`

Reply:
Changed to isLineInOrOut